profiles/kentik_snmp/a10_networks/a10-thunder.yml

```diff
@@ -45,6 +45,18 @@ metrics:
         OID: 1.3.6.1.4.1.22610.2.4.1.3.3.0
         poll_time_sec: 60
         tag: CPU
+      #Average A10 Load Balancer Average Control CPU Usag
+      - name: axSysAverageCtlCpuUsage
+        OID: 1.3.6.1.4.1.22610.2.4.1.3.4.0
+        poll_time_sec: 60
+      #Average A10 Load Balancer Average Data CPU Usag
+      - name: axSysAverageDataCpuUsage
+        OID: 1.3.6.1.4.1.22610.2.4.1.3.5.0
+        poll_time_sec: 60
```

Comment:
is it purposeful that you have set the polling interval for these 2 metrics at 60 seconds? Typically we reserve this only for the defined golden metrics to support anomaly detection (which expects at least 1 data point per minute in the New Relic platform).

there are multiple configuration options available in your own configuration file for `ktranslate` that allow you to override the profile settings here in case you need the 60s granularity. this allows you to meet your needs and protects the global profile that all users have from changes that impact ingest rates.